django_client_framework/api/related_model_api.py

```diff
@@ -116,7 +175,12 @@ def delete(self, request, *args, **kwargs):
                 f"Cannot remove {self.field_name} from {self.model.__name__} due to non-null constraints."
                 " Did you mean to delete the object directly?"
             )
-        self.field_val.remove(*self.__body_pk_ls)
+        selected = list(self.field_model.objects.filter(id__in=self.__body_pk_ls))
+        for selected_product in selected:
```

Comment:
`list` forces `.filter` to query the database. This is less efficient. It's better to use the QuerySet, right?